packages/edit-post/src/components/layout/index.js

```diff
@@ -99,7 +100,19 @@ function Layout( {
 				<div className="edit-post-layout__metaboxes">
 					<MetaBoxes location="advanced" />
 				</div>
+				{ isMobileViewport && sidebarIsOpened && <ScrollLock /> }
 			</div>
+			{ isRichEditingEnabled && mode === 'visual' && (
+				<div
+					className="edit-post-layout__footer"
```

Comment:
Nice, thank you for addressing it 👍 
I would appreciate some confirmation from the accessibility team. Although it's something that we can polish later as well.